src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -192,10 +192,18 @@ export async function promptForBoolean( message: string, initial: boolean ) {
 	return confirm.run();
 }
 
+const componentDisplayNames = {
+	wordpress: 'WordPress',
+	muPlugins: 'vip-go-mu-plugins',
+	jetpack: 'Jetpack',
+	clientCode: 'site-code',
+};
+
 export async function promptForComponent( component: string ) {
+	const componentDisplayName = componentDisplayNames[ component ];
```

Comment:
I suppose if an item is accidentally left off that list we'd want to default to the original?

Reply:
Good point. fixed